app/src/main/java/com/pvp/app/ui/screen/settings/SettingsScreen.kt

```diff
@@ -284,24 +282,19 @@ private fun SettingApplicationTheme(
 
 @Composable
 private fun SettingHealthConnectPermissions(context: Context) {
-    Button(
-        modifier = Modifier
-            .padding(
-                horizontal = 8.dp,
-                vertical = 4.dp
-            )
-            .fillMaxWidth(),
-        onClick = {
+    SettingCard(
+        title = "Health Connect",
+        description = "By enabling certain Health Connect permissions you are allowing us to provide you more great features. You can disable those permissions anytime by coming here.",
```

Comment:
Could this be split into few parts so that it wouldn't go out of the screen? Press `Enter` in certain spots, IDE will automatically split and join strings for you.